### ISSUE TITLE
App shortcut for "Block/unblock trackers" function

### ISSUE THREAD

**Issue by parcelcat** [MuntashirAkon/AppManager#1047]
"Block/unblock trackers" is the feature of App Manager that I use the most. It would be helpful if App Manager had an [app shortcut](https://developer.android.com/develop/ui/views/launch/shortcuts) for "Block/unblock trackers" so that the feature can be activated by long-pressing App Manager from the home screen or app drawer and tapping the app shortcut.

App shortcut example:

![App shortcut example](https://developer.android.com/static/images/guide/topics/ui/shortcuts.png)

**Comment by MuntashirAkon** [comment]
Duplicate of #660
